### DISCUSSION TITLE
How to simply execute a Command?

### DISCUSSION THREAD

**Discussion by BytePetter** [AvaloniaUI/Avalonia#11630] [Q&A]
The Tutorial for the Todo-List-App shows a method on how to execute Commands directly:

```
public void AddItem()
        {
            var vm = new AddItemViewModel();

            Observable.Merge(
                vm.Ok,
                vm.Cancel.Select(_ => (TodoItem)null))
                .Take(1)
                .Subscribe(model =>
                {
                    if (model != null)
                    {
                        List.Items.Add(model);
                    }

                    Content = List;
                });

            Content = vm;
        }

```
This, however, fails with the error of CS1660, as a Lambda-Expression cannot be converted into an 'IObserver<something>'.
That is, because .Subscribe() actually expects an IObserver, and the tutorial doesn't show of any other way of simply executing a command, which doesn't get a parameter from me or returns a result anyway. 

How can I simply execute one or multiple, Commands?

**Comment by jp2masa** [comment]
`System.Reactive` dependency was removed in #9749. If you don't care about the dependency problem it fixes, you just need to install `System.Reactive` in your project and it should work.

**Reply by maxkatz6** [reply]
It seems like ReactiveUI is still used in the code above, which means System.Reactive will be there.
In which case simple "using System;" should help.
